src/clj/rems/css/styles.clj

```diff
@@ -526,6 +526,14 @@
    [(s/> :.form-actions "*:not(:first-child)")
     (s/> :.commands "*:not(:first-child)")
     {:margin-left (u/em 0.5)}]
+   [".btn-opens-more::after" {:display "inline-block"
+                              :font-style "normal"
+                              :font-variant "normal"
+                              :text-rendering "auto"
+                              :-webkit-font-smoothing "antialiased"
+                              :font-family "'Font Awesome 5 Free'"
+                              :font-weight 900
+                              :content "'\\a0\\f141'"}] ; nbsp and ellipsis-h
```

Comment:
Hmm, isn't `fa-ellipsis-h` wrong here? We want the `...` and not the middle dots. Also I would put the `...` to the other place as well because the middle dots don't really work there so well.